<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonkeyPass</class>
 <widget class="QMainWindow" name="MonkeyPass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1276</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MonkeyPass</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="login_page">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="masterPassword"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="create_page">
       <widget class="QLineEdit" name="newPassword">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>106</y>
          <width>142</width>
          <height>50</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
       </widget>
       <widget class="QLineEdit" name="passwordConfirm">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>259</y>
          <width>142</width>
          <height>50</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="createButton">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>412</y>
          <width>80</width>
          <height>50</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
       <widget class="QPushButton" name="generateButton">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>565</y>
          <width>80</width>
          <height>50</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1276</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionImport_Enpass_File"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionImport_Enpass_File">
   <property name="text">
    <string>Import Enpass File</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
